java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -690,6 +690,200 @@ void testInnerJoinWithOnlyCommonKeys() {
     }
   }
 
+  @Test
```

Comment:
Can we have a test that verifies corner cases like `inputTable` is empty or `valuesTable` is empty?  I just want to be sure that it does the correct thing in those cases.

Reply:
If I am reading the code right- the bound result column is set to 0 in the case the input Table has no rows. I checked with a simple test and it return a column with one rows of value 0 (which does not help differentiate a legit 0 index insertion point except if the values table is nullable where it sets the result's validity buffer to indicate a returned result of null. I would not want to rely on values columns being nullable or not in order to get the right answer from an application's perspective. I will just throw an exception in the case either tables are empty. Let me know if you have any corrections here.